encryption_works.md

```diff
@@ -98,25 +98,58 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
 
 ## Anonymize Your Location with Tor
 
-Tor is a software service that allows you to use the Internet while concealing your IP address, which is, in general, a fairly accurate representation of your location. The [Tor](https://www.torproject.org/) network is made up of over 3,600 volunteer servers called nodes. When someone uses the Tor network to visit a website their connection gets bounced through three of these nodes (called a circuit) before finally exiting into the normal Internet. Anyone intercepting traffic will think your location is the final node which your traffic exits from.
+While your web habits can often be tracked when browsing the Internet, it's still possible to use the web and avoid many types of mass surveillance. The easiest way is to use the Tor Browser, a web browser like Firefox or Safari, but it allows you to browse the web while masking your IP address, which is generally an accurate representation of your approximate location. 
+
+The software is free and open-source, meaning that it comes with a high degree of trust, and is an adapted version of the Firefox browser.
+
+![The Tor Browser](https://raw.githubusercontent.com/tommycollison/encryption-works/master/images/tor_FPF.png)
+
+The Tor network is made up of over 6,000 volunteer servers, called nodes. When you use Tor, your connection is routed through three of these nodes — called a circuit – before exiting onto the normal Internet. Anyone intercepting Internet traffic will think your location is the final node which your traffic exits from.
+
+![To obscure your location, Tor routes your connection through three intermediaries.](/images/tor_relays.png)
+
+### Setup
+
+The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en). Once you download and install it, you'll see that it works like any other web browser but is a little bit slower, since your traffic is being routed through the Tor network to provide you with anonymity.
+
+When Snowden [was answering questions on Guardian's website](http://www.guardian.co.uk/world/2013/jun/17/edward-snowden-nsa-files-whistleblower) from a "secure Internet connection", he was probably routing his traffic through the Tor network. He may have also been using a [bridge](https://bridges.torproject.org/) to connect to the Tor network to make the fact that he was using Tor from his IP address less obvious to eavesdroppers.
+
+### Browsing Habits
+
+Here are some further tips to enhance your security and privacy while using the Tor browser. Some tips, such as disabling flash, are good advice for any web browser.
+
+- Turn on No Scripts for added security, as this prevents javascript from running in the browser, which could put you at risk of being identified.
```

Comment:
Instead of recommending they mess with NoScript directly (it is very easy to make the browser unusable by playing with NoScript without knowing what you're doing), we should recommend that they use the new [Security Slider](https://blog.torproject.org/blog/tor-browser-45-released).

We should recommend setting it to "Medium-High" for most people, which adds a lot of good security protections but does not fully disable Javascript. Note that for most people on most websites in this day and age, a web browser without javascript is a broken web browser, so we should only recommend that for the *most* security conscious users (e.g. like we do for sources using SecureDrop).